dmoj/executors/compiled_executor.py

```diff
@@ -138,33 +139,33 @@ def limit_executable():
         except ImportError:
             return None
 
-    def get_compile_process(self):
+    def get_compile_process(self) -> TimedPopen:
         kwargs = {'stderr': subprocess.PIPE, 'cwd': self._dir, 'env': self.get_compile_env(),
                   'preexec_fn': self.create_executable_limits(), 'time_limit': self.compiler_time_limit}
         kwargs.update(self.get_compile_popen_kwargs())
 
         return TimedPopen(self.get_compile_args(), **kwargs)
 
-    def get_compile_output(self, process):
+    def get_compile_output(self, process: TimedPopen) -> Tuple[bytes, bytes]:
```

Comment:
Oops, I think this is `bytes` after all.

Reply:
Not a tuple, just bytes?

Reply:
Also, are you referring to the Return value or `process`?

Reply:
The return value, yes. `safe_communicate` returns `Tuple[byes, bytes]` but it is indexed by `self.compile_output_index`.

Reply:
Ah, yes, I see it now